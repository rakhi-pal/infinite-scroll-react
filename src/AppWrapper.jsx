import App from './App';
import {createBrowserHistory} from 'history';

const AppWrapper = () => {
    const history = createBrowserHistory({forceRefresh:true});
    return (
        <App history={history} />
    )
}
export default AppWrapper;