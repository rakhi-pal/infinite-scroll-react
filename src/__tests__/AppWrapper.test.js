import { render, cleanup } from '@testing-library/react';
import AppWrapper from '../AppWrapper';

afterEach(cleanup);
test('renders App Wrapper component and tests snapshots', () => {
  const AppWrapperComponent = render(<AppWrapper />);
  expect(AppWrapperComponent).toMatchSnapshot();
});