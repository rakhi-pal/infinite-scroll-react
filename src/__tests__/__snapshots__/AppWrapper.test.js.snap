// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App Wrapper component and tests snapshots 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="login-wrapper"
      >
        <form
          class="login-form"
        >
          <div
            class="login-header"
          >
            <h2>
              Login
            </h2>
          </div>
          <div
            class="text-input"
          >
            <div
              class="MuiFormControl-root MuiTextField-root"
            >
              <label
                class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
                data-shrink="false"
                for="Username"
                id="Username-label"
              >
                Username
              </label>
              <div
                class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
              >
                <input
                  aria-invalid="false"
                  class="MuiInputBase-input MuiOutlinedInput-input"
                  id="Username"
                  type="text"
                  value=""
                />
                <fieldset
                  aria-hidden="true"
                  class="PrivateNotchedOutline-root-1 MuiOutlinedInput-notchedOutline"
                >
                  <legend
                    class="PrivateNotchedOutline-legendLabelled-3"
                  >
                    <span>
                      Username
                    </span>
                  </legend>
                </fieldset>
              </div>
            </div>
          </div>
          <div
            class="text-input"
          >
            <div
              class="MuiFormControl-root MuiTextField-root"
            >
              <label
                class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
                data-shrink="false"
                for="password"
                id="password-label"
              >
                Password
              </label>
              <div
                class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
              >
                <input
                  aria-invalid="false"
                  autocomplete="cc-csc"
                  class="MuiInputBase-input MuiOutlinedInput-input"
                  id="password"
                  type="password"
                  value=""
                />
                <fieldset
                  aria-hidden="true"
                  class="PrivateNotchedOutline-root-1 MuiOutlinedInput-notchedOutline"
                >
                  <legend
                    class="PrivateNotchedOutline-legendLabelled-3"
                  >
                    <span>
                      Password
                    </span>
                  </legend>
                </fieldset>
              </div>
              
            </div>
          </div>
          <div
            class="login-btn-container"
          >
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-contained login-btn MuiButton-containedPrimary"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                Login
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
          </div>
        </form>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="login-wrapper"
    >
      <form
        class="login-form"
      >
        <div
          class="login-header"
        >
          <h2>
            Login
          </h2>
        </div>
        <div
          class="text-input"
        >
          <div
            class="MuiFormControl-root MuiTextField-root"
          >
            <label
              class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
              data-shrink="false"
              for="Username"
              id="Username-label"
            >
              Username
            </label>
            <div
              class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
            >
              <input
                aria-invalid="false"
                class="MuiInputBase-input MuiOutlinedInput-input"
                id="Username"
                type="text"
                value=""
              />
              <fieldset
                aria-hidden="true"
                class="PrivateNotchedOutline-root-1 MuiOutlinedInput-notchedOutline"
              >
                <legend
                  class="PrivateNotchedOutline-legendLabelled-3"
                >
                  <span>
                    Username
                  </span>
                </legend>
              </fieldset>
            </div>
          </div>
        </div>
        <div
          class="text-input"
        >
          <div
            class="MuiFormControl-root MuiTextField-root"
          >
            <label
              class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
              data-shrink="false"
              for="password"
              id="password-label"
            >
              Password
            </label>
            <div
              class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
            >
              <input
                aria-invalid="false"
                autocomplete="cc-csc"
                class="MuiInputBase-input MuiOutlinedInput-input"
                id="password"
                type="password"
                value=""
              />
              <fieldset
                aria-hidden="true"
                class="PrivateNotchedOutline-root-1 MuiOutlinedInput-notchedOutline"
              >
                <legend
                  class="PrivateNotchedOutline-legendLabelled-3"
                >
                  <span>
                    Password
                  </span>
                </legend>
              </fieldset>
            </div>
            
          </div>
        </div>
        <div
          class="login-btn-container"
        >
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-contained login-btn MuiButton-containedPrimary"
            tabindex="0"
            type="button"
          >
            <span
              class="MuiButton-label"
            >
              Login
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </div>
      </form>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
